notebooks/GPU_Cluster_Sizing_with_NeMo_Agent_Toolkit.ipynb

```
@@ -0,0 +1,1164 @@
+    "    else:\n",
+    "        print(\"   source .venv/bin/activate\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Helper function to run nat commands with proper environment\n",
+    "def run_nat_command(args, **kwargs):\n",
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)